gads-ui/src/components/EmptyBlock/styles.module.scss

```diff
@@ -0,0 +1,116 @@
+.empty {
+    width: 100%;
+    margin-top: 2rem;
+    display: flex;
+    flex-direction: column;
+
+    align-items: center;
+    justify-content: center;
+
+    gap: 12px;
+
+    border: 1px solid var(--gray-700);
+    border-radius: 8px;
+
+    padding: 40px 32px 48px 32px;
+
+    -webkit-box-shadow: 0px 2px 5px 0px rgba(0,0,0,0.10);
+    -moz-box-shadow: 0px 2px 5px 0px rgba(0,0,0,0.10);
+    box-shadow: 0px 2px 5px 0px rgba(0,0,0,0.10);
+
+    img {
+        width: 160px;
+        opacity: 0.4;
+    }
+
+    h2 {
+        font-size: 16px;
+        font-weight: 600;
+
+        color: var(--gray-900);
+        width: 400px;
+
+        text-align: center;
+    }
+
+    span{
+        color: var(--gray-500);
+        font-size: 14px;
+
+        width: 400px;
+
+        text-align: center;
+    }
+
+    button {
```

Comment:
Are the styles below(and including) this obsolete? We dont have a button or a loading panel in the EmptyBox component?